src/Configuration/Config.NewtonsoftJson/test/Microsoft.Extensions.Configuration.NewtonsoftJson.Tests.csproj

```diff
@@ -5,12 +5,20 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\Config\test\Microsoft.Extensions.Configuration.Tests.csproj" />
-    <ProjectReference Include="..\..\testassets\Test.Common\Microsoft.Extensions.Configuration.Test.Common.csproj" />
+    <Compile Include="..\..\Config\test\ConfigurationProviderTestBase.cs">
+      <Link>test\ConfigurationProviderTestBase.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Config\testassets\Test.Common\ConfigurationProviderExtensions.cs">
```

Comment:
Again, please move these out of testassets/ since they're now used as shared source

Reply:
I have a feeling we'll be cleaning up a bunch of these things in a follow-up especially since I'll be removing a bunch of the shared sources that's already been duplicated in aspnetcore as well. I wouldn't block this PR on this.

Reply:
FYI I added an item to https://github.com/dotnet/aspnetcore/issues/19254 to make sure this isn't lost.